localstack/services/generic_proxy.py

```diff
@@ -381,7 +381,7 @@ def is_full_url(url):
         kwargs = {
             'method': method,
             'path': path,
-            'data': data_bytes,
+            'data': data,
```

Comment:
Thanks a lot for this PR @jthomas-root . This change may have larger implications across different parts of the code base. The idea here is to feed the original (unmodified) payload data to the post-invocation listener. Can we give it a try without this change - are the tests failing in that case?

Reply:
Thanks for the feedback, @whummer .  I don't believe any tests are failing, but I'll remove any changes to this method to be safe.  I'll look to recalculate the *key* and*location* in the `return_response` method for the s3 listener.